deploy/helm/elastic-agent/values.schema.json

```
@@ -331,6 +331,36 @@
                 }
             }
         },
+        "auto_ops": {
```

this is how it is inside values.yaml
```suggestion
        "autoOps": {
```